app/views/admin/order_cycles/checkout_options.html.haml

```diff
@@ -0,0 +1,82 @@
+= render partial: "/admin/order_cycles/order_cycle_top_buttons"
+
+- content_for :page_title do
+  = t :edit_order_cycle
+
+- payment_methods = order_cycle_distributors_payment_methods(@order_cycle)
+- shipping_methods = order_cycle_distributors_shipping_methods(@order_cycle)
+
+= form_for [main_app, :admin, @order_cycle], html: { class: "order_cycle" } do |f|
+
+  = render 'wizard_progress'
+
+  %fieldset.no-border-bottom
+    %legend{ align: 'center'}= t('.checkout_options')
+
+  = hidden_field_tag "order_cycle[selected_shipping_method_ids][]", ""
+
+  .row
+    .three.columns
+      &nbsp;
+    .ten.columns
+      %table.checkout-options
+        %thead
+          %tr
+            %th{ colspan: 2 }= t('.shipping_methods')
+          %tr{ "data-controller": "select-all" }
+            %td.text-center
+              %label
+                = check_box_tag "bla", nil, shipping_methods == @order_cycle.shipping_methods, { "data-action": "change->select-all#toggleAll", "data-select-all-target": "all" }
+                = "Select all"
+            %td
+              - if shipping_methods.any?
+                - shipping_methods.each do |shipping_method|
+                  %p
+                    %label{ class: ("disabled" unless shipping_method.frontend?) }
+                      = check_box_tag "order_cycle[selected_shipping_method_ids][]",
+                          shipping_method.id, @order_cycle.shipping_methods.include?(shipping_method),
+                          disabled: !shipping_method.frontend?,
+                          id: "order_cycle_selected_shipping_method_ids_#{shipping_method.id}",
+                          data: ({ "action" => "change->select-all#toggleCheckbox", "select-all-target" => "checkbox" } if shipping_method.frontend?)
+                      = shipping_method.name
+                      - unless shipping_method.frontend?
+                        = "(#{t('.back_end')})"
```

Comment:
We don't need to change this but just an idea how to give translators more flexibility and more clarity around what the context is:
```suggestion
                      - if shipping_method.frontend?
                        = shipping_method.name
                      - else
                        = t(".shipping_method_backend", name: shipping_method.name)
```

Reply:
:+1: good point